core/src/display_object/edit_text.rs

```diff
@@ -953,9 +959,8 @@ impl<'gc> EditText<'gc> {
         // If text's top is under the textbox's bottom, skip drawing.
         // TODO: FP actually skips drawing a line as soon as its bottom is under the textbox;
         //   Current logic is conservative for safety (and even of this I'm not 100% sure).
-        // TODO: we should also cull text that's above the textbox
-        //   (instead of culling, this can be implemented as having the loop start from `scrollY`th line)
         //   (maybe we could cull-before-render all glyphs, thus removing the need for masking?)
+        //   [KJ] FP always displays the first line, culls any line which is not fully visible
```

Comment:
What do you mean by "the first line" here? :thinking:

Reply:
Elaborated a bit about it in the comment, I mean the first visible line (taking into account vscroll).

Reply:
Yeah, that was what I guessed as only this makes logical sense to me; but the comment didn't clearly mean that to me - thank you for clarifying it!